src/core/l10n.js

```diff
@@ -17,7 +17,7 @@ const base = {
   editors_note: "Editor's note",
   editors: "Editors:",
   example: "Example",
-  fig: "Fig. ",
+  fig: "Figure ",
```

Comment:
I started using `figure`s in a specification, and the abbreviation seemed unhelpful.

Admittedly this is close to the bike shed...